src/main/java/mekanism/common/integration/multipart/MultipartMekanism.java

```diff
@@ -162,4 +164,28 @@ public static boolean hasConnectionWith(TileEntity tile, EnumFacing side)
 		
 		return Collections.emptyList();
 	}
+	
+	public static boolean isMultiPartContainer(TileEntity entity)
+	{
+		return entity instanceof IMultipartContainer;
+	}
+	
+	public static Collection<TileEntity> getMekanismTileEntities(TileEntity container)
+	{
+		ArrayList<TileEntity> entities = new ArrayList<>();
```

Comment:
You are correct, there will only ever be one `TileEntitySidedPipe` but there may be other Mekanism tile entities in the multipart (namely glow panels).

That being said after investigating this I've found a much better way of getting the appropriate tile entities by taking advantage of MCMP's own world wrapping so I'll be replacing this method with that.